ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Make sure you try the following before submitting the issue, thank you!
+
+- Check that umake is the latest version (16.11.1)
```

Comment:
I wouldn't specify any revision here for maintenance purpose but points to https://github.com/ubuntu/ubuntu-make/releases.
Tell they can check the version with `umake --version`